anybox/recipe/odoo/base.py

```diff
@@ -1667,6 +1671,17 @@ def finalize_addons_paths(self, check_existence=True):
                 assert os.path.isdir(path), (
                     "Not a directory: %r (aborting)" % path)
 
+        self.addons_paths = [
+            os.path.relpath(
+                addons_path,
+                os.path.join(
+                    self.odoo_dir,
+                    'openerp' if 'openerp' in addons_path else 'odoo'
+                ))
```

Comment:
This gives me `addons_path = addons,../addons` and, when trying to run Odoo, an error saying
```
FileNotFoundError: [Errno 2] No such file or directory: '/home/odoo/14.0/parts/addons'
```
In our current setup, we do not add this additional openerp/odoo part to the base directory when fetching the relpath: https://github.com/EssentNovaTeam/anybox.recipe.odoo/blob/master/anybox/recipe/odoo/base.py#L1633-L1634

If I leave this out, I get a buildout that allows me to run Odoo before and after relocation.

```
             os.path.relpath(
                 addons_path,
-                os.path.join(
-                    self.odoo_dir,
-                    'openerp' if 'openerp' in addons_path else 'odoo'
-                ))
+                self.odoo_dir,
+            )
```